docs/scenarios/nfs-to-ebs-pv/README.md

```diff
@@ -0,0 +1,4 @@
+## Migrating a **stateful** OpenShift app with **NFS PVs** to **Amazon EBS PVs**
+
+This scenario walks through Migration of a stateful OpenShift app whose Persistent Volume (PV) claims tie into an NFS server. Through the Migration process, the data stored on NFS will be moved onto [Amazon EBS (Elastic Block Storage)](https://aws.amazon.com/ebs/). 
+
```

Comment:
For nfs to ebs we support:
 - Copy, we will create a new PV backed by EBS on the destination side and copy data from the source NFS PV to the destination EBS PV.

vice-versa is supported, i.e. EBS to NFS backed PV.